utilities/pipelines/resourceRemoval/helper/Get-DependencyResourceNameList.ps1

```diff
@@ -19,15 +19,30 @@ function Get-DependencyResourceNameList {
     [CmdletBinding()]
     param (
         [Parameter(Mandatory = $false)]
-        [string] $DependencyParameterPath = (Join-Path (Split-Path (Split-Path $PSScriptRoot -Parent) -Parent) 'dependencies')
+        [string] $DependencyParameterPath = (Join-Path (Get-Item $PSScriptRoot).Parent.Parent.FullName 'dependencies')
     )
 
+    # Load used function
+    $repoRootPath = (Get-Item $PSScriptRoot).Parent.Parent.Parent.Parent.FullName
+    . (Join-Path $repoRootPath 'utilities' 'pipelines' 'tokensReplacement' 'Convert-TokensInParameterFile.ps1')
+
     $parameterFolders = Get-ChildItem -Path $dependencyParameterPath -Recurse -Filter 'parameters' -Directory
     $parameterFilePaths = [System.Collections.ArrayList]@()
     foreach ($parameterFolderPath in $parameterFolders.FullName) {
         $parameterFilePaths += Get-ChildItem -Path $parameterFolderPath -Recurse -Filter '*.json'
     }
 
+    $Settings = Get-Content -Path (Join-Path $repoRootPath 'settings.json') | ConvertFrom-Json
+    foreach ($parameterFilePath in $parameterFilePaths) {
+        $ConvertTokensInputs = @{
+            ParameterFilePath              = $parameterFilePath
```

Comment:
Which parameter file would this be? Will all dependency parameter files be converted?

Reply:
Yes, all